From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 26 Sep 2013 19:52:58 +0000 (+0200)
Subject: build-sys: since clock_gettime() moved to libc use mq_open to check for librt
X-Git-Url: http://git.0pointer.de/?p=rtkit.git;a=commitdiff_plain;h=d550837387e4185ebcde1023702580e35eb441e7

build-sys: since clock_gettime() moved to libc use mq_open to check for librt
---

diff --git a/configure.ac b/configure.ac
index 5a77363..62c17f5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -111,7 +111,7 @@ CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
 CC="$PTHREAD_CC"
 
 AC_SEARCH_LIBS([sched_setscheduler], [rt])
-AC_SEARCH_LIBS([clock_gettime], [rt])
+AC_SEARCH_LIBS([mq_open], [rt])
 AC_SEARCH_LIBS([cap_init], [cap])
 
 PKG_CHECK_MODULES(DBUS, dbus-1)
